docs/docs/the_aztec_network/creating_and_voting_on_proposals.md

```diff
@@ -0,0 +1,208 @@
+---
+sidebar_position: 4
+title: Governance and Proposal Process
+description: Learn how sequencers create and vote on protocol upgrade proposals through governance
+---
+
+## Overview
+
+This guide covers how protocol upgrades are performed through governance, including how sequencers signal support for payloads, how proposals are created, and how to participate in governance voting.
+
+## Prerequisites
+
+Before proceeding, you should:
+
+- Be familiar with running a sequencer node
+- Have a basic understanding of how governance works on Aztec
+
+## Proposal Lifecycle
+
+### Starting with a Payload
+
+All protocol upgrades consist of a series of commands executed on the protocol contracts or replacements of contract references. These steps are defined in a contract called a **payload** deployed on Ethereum.
+
+This guide assumes the payload already exists at a known address. You'll follow the payload's journey through the proposal, voting, and execution stages.
+
+:::warning Always Verify Payloads
+Always verify the payload address and contents via etherscan or similar block explorer before signaling support or voting. Carefully review what actions the payload will perform.
+:::
+
+A sample payload looks like this:
+
+```solidity
+contract UpgradePayload is IPayload {
+  IRegistry public immutable REGISTRY;
+  address public NEW_ROLLUP = address(new FakeRollup());
+
+  constructor(IRegistry _registry) {
+    REGISTRY = _registry;
+  }
+
+  function getActions() external view override(IPayload) returns (IPayload.Action[] memory) {
+    IPayload.Action[] memory res = new IPayload.Action[](1);
+
+    res[0] = Action({target: address(REGISTRY), data: abi.encodeWithSelector(REGISTRY.addRollup.selector, NEW_ROLLUP)});
+
+    return res;
+  }
+
+  function getURI() external pure override(IPayload) returns (string memory) {
+    return "UpgradePayload";
+  }
+}
+```
+
+If this payload's proposal passes governance voting, the governance contract will execute `addRollup` on the `Registry` contract.
+
+### Signaling Support for a Payload
+
+Sequencers must initiate all proposals through a signaling process. When you propose blocks as a sequencer, you signal support for a payload. Once enough signals accumulate, the payload qualifies to become an official proposal.
+
+#### How Signaling Works
+
+- Only the current block proposer for a slot can signal support during that slot
+- Your sequencer node automatically calls `signal` on the `GovernanceProposer` contract when proposing a block (if you've configured a payload address)
+- Rounds consist of 1000 slots each
+- At every 1000-block boundary, the system checks if any payload has received 750 or more signals (the quorum threshold)
```

Comment:
These parameters are different for testnet vs ignition. Can we make sure thats clear?